.github/workflows/test.yml

```
@@ -294,9 +294,48 @@ jobs:
 
+  Package-Lock-Validation:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code with full history
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0 # Needed to diff against base branch/commit history
+
+      - name: Get changed package-lock.json files
+        id: changed-files
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:17:35: Double quote to prevent globbing and word splitting

(shellcheck)